server.js

```diff
@@ -1,3 +1,15 @@
 import app from './app.js';
+import { connectToDB } from './connections.js';
 
-app.listen({ port: process.env.PORT || 3000 });
+function loaders() {
+  try {
+    connectToDB();
+    app.listen({ port: process.env.PORT || 3000 });
+  } catch (error) {
+    app.logger(error);
+    loaders();
```

Comment:
why `loaders` here?
This can make infinate loop

Reply:
removed